src/zcl_abapgit_objects.clas.abap

```diff
@@ -20,6 +20,29 @@ CLASS zcl_abapgit_objects DEFINITION
         files TYPE zif_abapgit_definitions=>ty_files_tt,
         item  TYPE zif_abapgit_definitions=>ty_item,
       END OF ty_serialization .
+    TYPES:
+      ty_deserialization_step TYPE string.
+    TYPES:
+      ty_deserialization_step_tt TYPE STANDARD TABLE OF ty_deserialization_step
+                                            WITH DEFAULT KEY .
+    TYPES:
+      BEGIN OF ty_step_data,
+        id           TYPE string,
```

Comment:
should be type `ty_deserialization_step`?

Reply:
fixed